windows/src/desktop/setup/UfrmInstallOptions.pas

```diff
@@ -260,23 +260,10 @@ procedure TfrmInstallOptions.SetupDynamicOptions;
     if FInstallInfo.IsInstalled
       then Text := FInstallInfo.Text(ssOptionsUpgradeKeyman)
       else Text := FInstallInfo.Text(ssOptionsInstallKeyman);
-
-{
-    if not FInstallInfo.IsInstalled then
-      case FInstallInfo.BestMsi.LocationType of
-        iilLocal:  Text := FInstallInfo.Text(ssOptionsInstallKeyman, [FInstallInfo.BestMsi.Version]);
-        iilOnline: Text := FInstallInfo.Text(ssOptionsDownloadInstallKeyman, [FInstallInfo.BestMsi.Version, FormatFileSize(FInstallInfo.BestMsi.Size)]);
-      end
-    else
-      case FInstallInfo.BestMsi.LocationType of
-        iilLocal:  Text := FInstallInfo.Text(ssOptionsUpgradeKeyman, [FInstallInfo.BestMsi.Version]);
-        iilOnline: Text := FInstallInfo.Text(ssOptionsDownloadUpgradeKeyman, [FInstallInfo.BestMsi.Version, FormatFileSize(FInstallInfo.BestMsi.Size)]);
-      end;
-}
   end
   else if FInstallInfo.IsInstalled then
   begin
-    Text := FInstallInfo.Text(ssOptionsKeymanAlreadyInstalled, [FInstallInfo.InstalledVersion.Version]);
+    Text := FInstallInfo.Text(ssOptionsKeymanAlreadyInstalled, [FInstallInfo.InstalledVersion.Version]);//TODO VersionWithTag?
```

Comment:
This `//TODO` is intentionally left unresolved?

Reply:
Yes, because we don't have access to `VersionWithTag` without a lot more plumbing here 😭. So this is left as a future (low priority) project.